assets/src/edit-story/utils/getAdjustedElementDimensions.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Internal dependencies
+ */
+import { PAGE_WIDTH } from '../constants';
+
+/**
+ * Updates Text element's width and height if it's being resized from edges or there are font changes.
+ *
+ * @param {Object}  element Element to measure.
+ * @param {string}  content Text content.
```

Comment:
`warning  @param "content" does not match an existing function parameter  jsdoc/check-param-names`

Reply:
Yep, I saw that, I don't know how to resolve this since it actually does match 😕(or am I seeing incorrectly/missing something?)

Reply:
The issue is there are not params you are passing. You are passing an object that you are then deconstructing. So the comment should look like this. 
```
 * @param {Object}  object Object.
 * @param {Object}  object.element Element to measure.
 * @param {string}  object.content Text content.
 * @param {number}  object.width   Current element width.
 * @param {number}  object.height  Current element height.
 * @param {boolean} object.fixedMeasure If one side is locked for changing automatically, can be 'width' or 'height'.
```

Reply:
Had the same issue [here](https://github.com/ampproject/amp-wp/pull/4122/files#diff-22d2f01487a6347a34a325d99c905985R34).

Reply:
Thanks! Fixed.